.pre-commit-config.yaml

```diff
@@ -143,6 +143,13 @@ repos:
             |static/app/utils/profiling/profile/formats/typescript/trace\.json
           )$
 
+      - id: knip
+        name: knip
+        language: system
+        files: \.(ts|js|tsx|jsx|mjs)$
+        stages: [pre-push]
+        entry: bash -c 'if [ -n "${SENTRY_KNIP_PRE_PUSH:-}" ]; then exec ./node_modules/.bin/knip; fi' --
+
```

Comment:
if this hook does not operate on files it should have `pass_filenames: false`

Reply:
done: 221cefd3